migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Do readmes also need new lines at the end?

Reply:
Don't know :thinking: